testbed/Makefile

```diff
@@ -22,4 +27,9 @@ run-stability-tests:
 
 .PHONY: install-tools
 install-tools:
-	go install github.com/jstemmer/go-junit-report
+	@if [ $(GO_MAJOR_VERSION) -eq 1 ] && [ $(GO_MINOR_VERSION) -le 15 ]; then \
+	go install github.com/jstemmer/go-junit-report; \
+	else \
+	go install github.com/jstemmer/go-junit-report@latest;  \
+	fi
+
```

Comment:
I think this should be installed via the main Makefile.install-tools, we already have that there? Probably just delete this? or reference the same go.mod for the junit version?

```
	cd $(TOOLS_MOD_DIR) && go install github.com/jstemmer/go-junit-report
```

Reply:
Sure thing - will remove